server/services/formService.js

```diff
@@ -295,6 +295,25 @@ module.exports = function createFormService(formClient) {
     }
   }
 
```

Comment:
We could merge setAwaitingApproval and backToCategoriser to a general purpose status change function maybe at some point